From 62f350064c5e05e71775c16eec9fa93dbb78d660 Mon Sep 17 00:00:00 2001
From: Wenlin Kang <wenlin.kang@windriver.com>
Date: Sun, 1 Dec 2024 22:43:11 -0800
Subject: [PATCH 1/3] debian/lintian: silence lintian error
 systemd-service-file

Silence lintian error:
E: dracut-core: service-file-is-not-a-file [lib/systemd/system/dracut-cmdline.service]
E: dracut-core: service-file-is-not-a-file [lib/systemd/system/dracut-initqueue.service]
E: dracut-core: service-file-is-not-a-file [lib/systemd/system/dracut-mount.service]
E: dracut-core: service-file-is-not-a-file [lib/systemd/system/dracut-pre-mount.service]
E: dracut-core: service-file-is-not-a-file [lib/systemd/system/dracut-pre-pivot.service]
E: dracut-core: service-file-is-not-a-file [lib/systemd/system/dracut-pre-trigger.service]
E: dracut-core: service-file-is-not-a-file [lib/systemd/system/dracut-pre-udev.service]
E: dracut-core: service-file-is-not-a-file [lib/systemd/system/dracut-shutdown-onfailure.service]
E: dracut-core: service-file-is-not-a-file [lib/systemd/system/dracut-shutdown.service]
E: dracut-core: service-file-is-not-a-file [lib/systemd/system/initrd.target.wants/dracut-cmdline.service]
E: dracut-core: service-file-is-not-a-file [lib/systemd/system/initrd.target.wants/dracut-initqueue.service]
E: dracut-core: service-file-is-not-a-file [lib/systemd/system/initrd.target.wants/dracut-mount.service]
E: dracut-core: service-file-is-not-a-file [lib/systemd/system/initrd.target.wants/dracut-pre-mount.service]
E: dracut-core: service-file-is-not-a-file [lib/systemd/system/initrd.target.wants/dracut-pre-pivot.service]
E: dracut-core: service-file-is-not-a-file [lib/systemd/system/initrd.target.wants/dracut-pre-trigger.service]
E: dracut-core: service-file-is-not-a-file [lib/systemd/system/initrd.target.wants/dracut-pre-udev.service]
E: dracut-core: service-file-is-not-a-file [lib/systemd/system/sysinit.target.wants/dracut-shutdown.service]

In previous commit 5758070954831072438dd9061febc332072057b1, it has
fixed one, but some are still existing, this patch fixes them.

Signed-off-by: Wenlin Kang <wenlin.kang@windriver.com>
---
 debian/dracut-core.lintian-overrides | 17 +++++++++++++++++
 1 file changed, 17 insertions(+)

diff --git a/debian/dracut-core.lintian-overrides b/debian/dracut-core.lintian-overrides
index 3e612dfe..06756bdb 100644
--- a/debian/dracut-core.lintian-overrides
+++ b/debian/dracut-core.lintian-overrides
@@ -2,3 +2,20 @@
 dracut-core: executable-in-usr-lib [usr/lib/dracut/*]
 # These services run in the initrd and are allowed to linger until the very end of shutdown sequence
 dracut-core: systemd-service-file-shutdown-problems [usr/lib/systemd/system/*dracut-*.service]
+dracut-core: service-file-is-not-a-file [lib/systemd/system/dracut-cmdline.service]
+dracut-core: service-file-is-not-a-file [lib/systemd/system/dracut-initqueue.service]
+dracut-core: service-file-is-not-a-file [lib/systemd/system/dracut-mount.service]
+dracut-core: service-file-is-not-a-file [lib/systemd/system/dracut-pre-mount.service]
+dracut-core: service-file-is-not-a-file [lib/systemd/system/dracut-pre-pivot.service]
+dracut-core: service-file-is-not-a-file [lib/systemd/system/dracut-pre-trigger.service]
+dracut-core: service-file-is-not-a-file [lib/systemd/system/dracut-pre-udev.service]
+dracut-core: service-file-is-not-a-file [lib/systemd/system/dracut-shutdown-onfailure.service]
+dracut-core: service-file-is-not-a-file [lib/systemd/system/dracut-shutdown.service]
+dracut-core: service-file-is-not-a-file [lib/systemd/system/initrd.target.wants/dracut-cmdline.service]
+dracut-core: service-file-is-not-a-file [lib/systemd/system/initrd.target.wants/dracut-initqueue.service]
+dracut-core: service-file-is-not-a-file [lib/systemd/system/initrd.target.wants/dracut-mount.service]
+dracut-core: service-file-is-not-a-file [lib/systemd/system/initrd.target.wants/dracut-pre-mount.service]
+dracut-core: service-file-is-not-a-file [lib/systemd/system/initrd.target.wants/dracut-pre-pivot.service]
+dracut-core: service-file-is-not-a-file [lib/systemd/system/initrd.target.wants/dracut-pre-trigger.service]
+dracut-core: service-file-is-not-a-file [lib/systemd/system/initrd.target.wants/dracut-pre-udev.service]
+dracut-core: service-file-is-not-a-file [lib/systemd/system/sysinit.target.wants/dracut-shutdown.service]
-- 
2.42.0

